frontend/src/components/github-readme.css

```
@@ -0,0 +1,278 @@
+
+.github-readme-body.markdown-body table th,
+.github-readme-body.markdown-body table td {
+    padding: 0.75rem 1rem;
+    vertical-align: top;
+}
+
+.github-readme-body.markdown-body code,
+.github-readme-body.markdown-body pre,
+.github-readme-body.markdown-body kbd,
+.github-readme-body.markdown-body samp {
+    font-family: ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, monospace;
```

Stylelint (17.5.0)

[error] Expected quotes around "SF Mono" (font-family-name-quotes)

(font-family-name-quotes)